git-cliff/src/lib.rs

```diff
@@ -797,3 +812,40 @@ pub fn run_with_changelog_modifier(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
The tests should be named more properly. Maybe use the name of the module as a prefix (`path_detects_*` or `util_detects_*`, etc.)